.github/ISSUE_TEMPLATE/feature_graduation.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature graduation
+about: Plan the graduation of an experimental feature
+title: 'Graduate XXX feature'
+labels: ["feature_graduation"]
+projects: ["semantic-kernel"]
+assignees: ''
+
+---
+
+---
+name: Feature graduation
+about: Plan the graduation of an experimental feature
+
+---
+
+Checklist to be completed when graduating an experimental feature
+
+- [ ] Contact Matthew for list of sample use cases
+- [ ] Verify there are sample implementations​ for each of the use cases
+- [ ] ​Verify API docs are complete and arrange to have them published
+- [ ] Make appropriate updates to Learn docs​
+- [ ] Make appropriate updates to Concept samples
+- [ ] Verify there are no serious open Issues​​
+- [ ] Update table in EXPERIMENTS.md
+- [ ] Remove SKEXP​ flag from the experimental code
```

Comment:
Should also have an item for releasing a blog and reviewing with the other languages

Reply:
And telemetry